x-pack/plugin/index-lifecycle/src/main/java/org/elasticsearch/xpack/indexlifecycle/action/RestGetOperationModeStatusAction.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+package org.elasticsearch.xpack.indexlifecycle.action;
+
+import org.elasticsearch.client.node.NodeClient;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.rest.BaseRestHandler;
+import org.elasticsearch.rest.RestController;
+import org.elasticsearch.rest.RestRequest;
+import org.elasticsearch.rest.action.RestToXContentListener;
+import org.elasticsearch.xpack.core.indexlifecycle.action.GetOperationModeStatusAction;
+import org.elasticsearch.xpack.indexlifecycle.IndexLifecycle;
+
+public class RestGetOperationModeStatusAction extends BaseRestHandler {
```

Comment:
I think we should call this API `*GetStatusAction` since that is the endpoint that it maps to

Reply:
agreed, another one of case of "crossed my mind, but decided against it for mediocre reasons". Will change